contrib/mixin/test.yaml

```diff
@@ -121,7 +121,7 @@ tests:
   - interval: 1m
     input_series:
       - series: 'etcd_mvcc_db_total_size_in_bytes{job="etcd",instance="10.10.10.0"}'
-        values: '0 1 2 0 0 1 0 3 0 0 0 0 0 0 0 0'
+        values: '0 1 2 2 2 3 3 6 6 6 6 6 6 6 6 6'
```

Comment:
we should leave these returns-to-zero in place.  The issue with `increase` is that it unrolls 0 1 2 0 0 1 to 0 1 2 2 2 3 and thinks "aah, this thing is taking off!".  When in reality, the size going 0 1 2 0 0 1 is "actually dropped to half of its previous peak; fantastic!".  Showing that we don't alert in that case demonstrates the fix that moving away from `increase` is delivering.  We should have a new test with 0 1 2 2 2 3 3 6 6 6 6 6 6 6 6 6 to show that when DB size is actually increasing like that, the alert continues to fire.

Although really... it's still only 6 bytes?  Is that really a situation where we want the alert firing, regardless of how quickly it increased to that point?  Perhaps we can add a floor where we need recent growth that's a significant chunk of the available DB capacity (`etcd_server_quota_backend_bytes`) instead of growth that's a significant chunk of the previous DB consumption?

Reply:
I think `predict_linear` will be better in this case, since it will predict the size of the database in the next hours, based on the increase size in the previous hours 

cc @ahrtr